### ISSUE TITLE
COA git resource get (rebase) on secrets fails

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/paas-templates#731]
### Expected behavior

As a paas-templates operator, in order to have robust and predictable system, I expect concurrency of COA job execution to not intefere

### Observed behavior

Sometimes, jobs fails in the git fetch, a manual restart of the job fixes the problems. This particularly reproduce when jobs concurrently execute (e.g. feature branch which simultaneously updates all of the osb-cmdb instances)

``` 
rebasing...
From https://elpaaso-gitlab.redacted-domain/paas-template/secrets
 * branch              master     -> FETCH_HEAD
 * [new branch]        master     -> push-target/master
Successfully rebased and updated refs/heads/master.

remote: error: cannot lock ref 'refs/heads/master': is at 00d0c0f5b7bec9a5b7912b2931b8aaa9bb340216 but expected 45c783450a12c9a2af2ba7c4b129011fd5952081        
To https://elpaaso-gitlab.redacted-domain/paas-template/secrets.git
 ! [remote rejected]   HEAD -> master (failed to update ref)

error: failed to push some refs to 'https://elpaaso-gitlab.redacted-domain/paas-template/secrets.git'
failed with non-rebase error
```

This was reproduced on a get of a git resource, see redacted screenshot below
![image](https://user-images.githubusercontent.com/4748380/77165814-58385b00-6ab3-11ea-9734-a729c1b004d7.png)

As a comparison, here is a redacted screenshot of a successful case
![image](https://user-images.githubusercontent.com/4748380/77165604-eeb84c80-6ab2-11ea-9aa6-ac1ae03f5c56.png)

/CC @JSimon-Orange 

### Affected release

Reproduced on version v45

**Comment by o-orand** [comment]
This is fixed in COA 4.3.1 by adding a retry  ![RetryOnPush](https://user-images.githubusercontent.com/976115/77434299-f04e8100-6de0-11ea-9508-286131568a22.png)

**Comment by poblin-orange** [comment]
morover, the micro-depls/gitlab has been scaled up, and gitlab woker tuned according to gitlab ce sizing guides (number of worker / nbr of vcpu).
Repopen of this occurs again
